contrib/basicauth/nginx/nginx.conf

```diff
@@ -0,0 +1,27 @@
+user  nginx;
```

Comment:
Let's put a note here that the parameters are for demonstration purposes. Users may need to tune them specifically for their environment. Maybe link out the nginx docs.